tasks/gulp/compile-js.js

```diff
@@ -56,3 +56,25 @@ gulp.task('js:compile-all-govuk-and-hmrc', () => gulp.src([
   .pipe(eol())
   .pipe(sourcemaps.write('./maps'))
   .pipe(gulp.dest(destinationPath)));
+
+gulp.task('js:compile-accessible-autocomplete', () => gulp.src([`${configPaths.src}accessible-autocomplete.js`])
+  .pipe(rollup({
+    plugins: [
+      babel(),
+      resolve(),
+      commonjs(),
+    ],
+  }, {
+    name: 'GOVUKAccessibleAutocomplete',
```

Comment:
With this, I wonder if we should make it match the govuk version, I'm thinking the prefix might confuse people using the documentation for accessible autocomplete, where it's ["accessibleAutocomplete"](https://github.com/alphagov/accessible-autocomplete/blob/main/webpack.config.babel.js#L115)

Reply:
@oscarduignan Interesting question. For me, the main reason to `GOVUKAccessibleAutocomplete` would be that it makes distinct our JS / assets - at this time we happen to be using the `alphagov/accessible-autocomplete`, BUT if that library gets replaced, or we decide to swap in another implementation, our naming could end up being not aligned with our convention elsewhere but also not aligned with any other library.

On the other hand, it would be a simpler upgrade for teams to use the same library naming.

On balance though, my preference would be to name the import as `GOVUKAccessibleAutocomplete` as it's less coupled (if only by naming convention) to a specific implementation.

Reply:
@oscarduignan As discussed, this has been renamed `HMRCAccessibleAutocomplete` so that it is clear that our implementation may be different to the GDS implementation (at the moment it is the same, but we don't want to couple the naming).